test/benchmark/llvm_benchmark.cpp

```diff
@@ -36,14 +41,14 @@ void LLVMBenchmark::generate_llvm(const std::shared_ptr<ast::Program>& node) {
     logger->info("Created LLVM IR module from NMODL AST in {} sec", diff.count());
 }
 
-void LLVMBenchmark::run_benchmark(const std::shared_ptr<ast::Program>& node) {
+void LLVMBenchmark::run_benchmark_on_cpu(const std::shared_ptr<ast::Program>& node) {
     // Set the codegen data helper and find the kernels.
     auto codegen_data = codegen::CodegenDataHelper(node, llvm_visitor.get_instance_struct_ptr());
     std::vector<std::string> kernel_names;
     llvm_visitor.find_kernel_names(kernel_names);
 
     // Get feature's string and turn them off depending on the cpu.
-    std::string cpu_name = cpu == "default" ? llvm::sys::getHostCPUName().str() : cpu;
+    const auto cpu_name = platform.get_name();
```

Comment:
Note that `get_name()` can return "default" if user has't specified the target CPU. So I guess we should have:
```
const auto cpu_name  platform.get_name() == "default" ? llvm::sys::getHostCPUName().str() : platform.get_name();
```